recipes/gtkmm/2.x.x/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+class GtkmmConan(ConanFile):
+    name = "gtkmm"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "LGPL-2.1-or-later"
+    homepage = "https://www.gtk.org"
+    description = "The C++ API for GTK."
+    settings = {"os": "Linux"}
+    options = {"version": [2]}
+    default_options = {"version": 2}
```

Comment:
This option depends on the value `self.options["gtk"].version`, and is thus not choose-able freely.
Having `-o gtk:version=2 -o gtkmm:version=3` is an invalid configuration.

Also, I don't see you using this gtk version value anywhere in the recipe.
How does it choose between gtk-2 and gtk-3?

I would do the following:
```suggestion
    options = {"version": "auto"}
    default_options = {"version": "auto"}

    @property
    def _gtk_version(self):
        return self.options["gtk"].version

    def package_id(self):
        self.info.options.version = self._gtk_version
```
and use `self._gtk_version` anywhere.